docs/microscope-installation-guide.md

```diff
@@ -7,6 +7,18 @@ This guide will walk through a complete recOrder installation consisting of:
 
 Before you start you will need a machine with Windows 10, a Meadowlark DS5020 connected to a liquid crystal device, and a microscope system compatible with `Micromanager`. 
 
+## Install Meadowlark DS5020 and liquid crystals
+
+**New installations:** start by installing the Meadowlark DS5020 and liquid crystals using the software on the USB stick provided by Meadowlark. You will need to install USB drivers and CellDrive5000 for compatibility with `recOrder-napari==0.4.0`.
```

Comment:
The USB sticks carry a label. One of our latest ones says "CellDrive 5000 v1.07 3/17/20". We should specify that the software version should be >=1.08 and that we've tested with v 1.08. This specific version can be requested from Meadowlark, I think, but hopefully that won't be needed. The intention of distributing the MLO device adapter with MM was that it would always work with the stick shipped from MLO. If we find that it breaks in the future, we should upgrade the libraries in the MM `3rdparty` repo to match the new MLO USB driver dlls

Reply:
It also doesn't hurt to add the Firmware>1.01 since they will be checking for the software version. The Firmware version also is the one that pops up when you add the device to a new config.

Reply:
All sounds good...I've improved the wording and specifically said that we've tested against software 1.08 and firmware 1.04.